pytorch_optimizer/lr_scheduler/chebyshev.py

```diff
@@ -18,7 +18,7 @@ def get_chebyshev_steps(num_epochs: int, small_m: float = 0.05, big_m: float = 1
     c, r = (big_m + small_m) / 2.0, (big_m - small_m) / 2.0
     thetas = (np.arange(num_epochs) + 0.5) * np.pi / num_epochs  # epoch starts from 0, so +0.5 instead of -0.5
 
-    return 1.0 / (c - r * np.cos(thetas))
+    return 1.0 / (c - r * np.cos(thetas) + 1e-8)
```

Comment:
I think `c - r * np.cos(thetas)` cannot be zero.

* c = `(big_m + small_m) / 2`
* r = `(big_m - small_m) / 2`

suppose, `c - r * np.cos(thetas) = 0`
* `c = r * np.cos(thetas)`
* `c / r = np.cos(thetas)`
* `(big_m + small_m) / (big_m - small_m) = np.cos(thetas)`

where `big_m` > `small_m` > 0, then, left side always `> 1.0`. however, the range of `np.cos(thetas)` is `[-1, 1]`.

meaning that, `c - r * np.cos(thetas)` always positive.

in conclusion, we don't need to add the epsilon here too!